test/task.jl

```diff
@@ -0,0 +1,5 @@
+@testset "Submit task" begin
+    oid = submit_task()
+    pid = String(take!(get(oid)))
+    @test all(isdigit, pid)
```

Comment:
can you additionally test taht teh PID is _different_ than the test runner/driver PID?